src/SourceGenerators/Uno.UI.SourceGenerators.Tests/XamlCodeGeneratorTests/Out/TTW/XamlCodeGenerator_MainPage_d6cd66944958ced0c513e0a04797b51d.cs

```diff
@@ -0,0 +1,219 @@
+			))
+			.GenericApply(((c3) => 
+			{
+				/* _isTopLevelDictionary:False */
+				__that._component_1 = c3;
+				// Class TestRepro.MainPage
+				global::Uno.UI.ResourceResolverSingleton.Instance.ApplyResource(c3, global::Windows.UI.Xaml.Controls.Page.BackgroundProperty, "ApplicationPageBackgroundThemeBrush", isThemeResourceExtension: true, isHotReloadSupported: false, context: global::MyProject.GlobalStaticResources.__ParseContext_);
+				global::Uno.UI.FrameworkElementHelper.SetBaseUri(c3, __baseUri_MainPage_d6cd66944958ced0c513e0a04797b51d);
+				c3.CreationComplete();
+			}
+			))
+			;
+			OnInitializeCompleted();
+
+			Bindings = new MainPage_Bindings(this);
+			Loading += (s, e) => 
+			{
+				__that.Bindings.Update();
+				__that.Bindings.UpdateResources();
+			}
+			;
+		}
+		partial void OnInitializeCompleted();
+		private global::Windows.UI.Xaml.Markup.ComponentHolder _component_0_Holder  = new global::Windows.UI.Xaml.Markup.ComponentHolder(isWeak: true);
+		private global::Windows.UI.Xaml.Controls.TextBlock _component_0
+		{
+			get
+			{
+				return (global::Windows.UI.Xaml.Controls.TextBlock)_component_0_Holder.Instance;
+			}
+			set
+			{
+				_component_0_Holder.Instance = value;
+			}
+		}
+		private global::Windows.UI.Xaml.Markup.ComponentHolder _component_1_Holder  = new global::Windows.UI.Xaml.Markup.ComponentHolder(isWeak: true);
+		private global::Windows.UI.Xaml.Controls.Page _component_1
+		{
+			get
+			{
+				return (global::Windows.UI.Xaml.Controls.Page)_component_1_Holder.Instance;
+			}
+			set
+			{
+				_component_1_Holder.Instance = value;
+			}
+		}
+		private interface IMainPage_Bindings
+		{
+			void Initialize();
+			void Update();
+			void UpdateResources();
+			void StopTracking();
+		}
+		#pragma warning disable 0169 //  Suppress unused field warning in case Bindings is not used.
+		private IMainPage_Bindings Bindings;
+		#pragma warning restore 0169
+		[global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
+		private class MainPage_Bindings : IMainPage_Bindings
+		{
+			#if UNO_HAS_UIELEMENT_IMPLICIT_PINNING
+			private global::System.WeakReference _ownerReference;
+			private global::TestRepro.MainPage Owner { get => (global::TestRepro.MainPage)_ownerReference?.Target; set => _ownerReference = new global::System.WeakReference(value); }
+			#else
+			private global::TestRepro.MainPage Owner { get; set; }
+			#endif
+			public MainPage_Bindings(global::TestRepro.MainPage owner)
+			{
+				Owner = owner;
+			}
+			void IMainPage_Bindings.Initialize()
+			{
+			}
+			void IMainPage_Bindings.Update()
+			{
+				var owner = Owner;
+				owner._component_0.ApplyXBind();
+			}
+			void IMainPage_Bindings.UpdateResources()
+			{
+				var owner = Owner;
+				owner._component_0.UpdateResourceBindings();
+				owner._component_1.UpdateResourceBindings();
+			}
+			void IMainPage_Bindings.StopTracking()
+			{
+			}
+		}
+		private bool TryGetInstance_xBind_1(global::TestRepro.MainPage ___tctx, out object o)
```

Comment:
Could those methods be static?

Reply:
@jeromelaban Good catch! I think this can be static indeed!